proxy.go

```diff
@@ -351,9 +351,20 @@ func (rp *reverseProxy) serveFromCache(s *scope, srw *statResponseWriter, req *h
 		if err != nil {
 			err = fmt.Errorf("%s: %w; query: %q", s, err, q)
 			respondWith(srw, err, http.StatusInternalServerError)
-			return
 		}
 	} else {
+		// Do not cache responses greater than max payload size.
+		if contentLength > int64(s.user.cache.MaxPayloadSize) {
+			cacheSkipped.With(labels).Inc()
+			log.Debugf("%s: Request will not be cached. Content length (%d) is greater than max payload size (%d)", s, contentLength, s.user.cache.MaxPayloadSize)
```

Comment:
I think it's worth to place it as info log

Reply:
Done